### ISSUE TITLE
Buzzing sound after clicking Apple II speaker

### ISSUE THREAD

**Issue by 6502-workshop** [AppleWin/AppleWin#534]
Thanks for the upgrades in this version!

I fired it up today and the noticed that there is a trailing "buzz" like sound after doing anything in a game that makes a sound (example, walking around in Ultima V). Additionally, it occurs in conjunction with the Apple II "beep" sound that plays when clicking the Apple button in the upper right corner.

**Comment by tomcw** [comment]
Thanks for trying the new version!

What Windows version are you running on?
What version of AppleWin _were_ you using before (which presumably didn't have this issue)?
Are you able to reproduce with older AppleWin versions? (eg. 1.25.0.4 and 1.26.x)

When you press the Apple button (upper right corner) the ROM does the speaker beep, then runs firmware at $C600 which quickly turns the driver motor on (activating full-speed... assuming you are in the default disk mode = "Enhanced Speed"). You can get a bit of buzz(*) with speaker sound when full speed is enabled/disabled.

Can you try setting disk mode = "Authentic Speed"?

(*) I get it for older versions of AppleWin too, eg. 1.25.0.4 and 1.26.x.

**Comment by 6502-workshop** [comment]
Hi Tom,

Thanks for the follow-up.

I noticed the issue on Windows 7. The issue doesn't occur for me on
AppleWIN_1.26.2.2 or 1.25.04. I tried authentic speed (on 1.27) and the
issue still occurs.

However, I just realized that I was in an RDP session between my Mac and PC
when I noticed this. I tried 1.27 on my PC without the RDP session and the
issue doesn't occur. I've noticed a variety of sound issues in RDP sessions
with other applications, and I'm running a beta version of RDP so this
probably isn't worth digging into. Sorry for the false alarm. I use RDP so
much when I'm doing development on my PC that I forget I'm in it sometimes.


Mark
